cmd/server/main.go

```diff
@@ -87,6 +87,8 @@ func run(args []string) error {
 		azStorageAccount           = fs.String("secret_azure_storage_account", "", "The storage account to authenticate against for blob operations")
 		azSourcePortfolioContainer = fs.String("secret_azure_source_portfolio_container", "", "The container in the storage account where we write raw portfolios to")
 
+		azEventWebhookSecret = fs.String("secret_azure_webhook_secret", "", "The shared secret required for incoming webhooks")
```

Comment:
IIUC, the structure here will make key rotation hard without failing a few requests in there, since we have exactly one secret we consider valid. Consider bifurcating this into "what auth secret should be set" and "which auth secret**s** should be accepted" to at least allow for the possibility of rotation sans downtime.

Reply:
An excellent suggestion, I'll send you a follow-up today.

Reply:
https://github.com/RMI-PACTA/app/pull/53